var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -194,7 +194,6 @@ class RocmOpenmpExtras(Package):
         depends_on("hsakmt-roct@" + ver, when="@" + ver)
         depends_on("comgr@" + ver, when="@" + ver)
         depends_on("hsa-rocr-dev@" + ver, when="@" + ver)
-        depends_on("llvm-amdgpu@{0} ~openmp".format(ver), when="@" + ver)
```

Comment:
This needs to be added back. rocm-openmp-extras is an extension to the llvm-amdgpu package.